### PR TITLE
Add configs for Statistics API

### PR DESCRIPTION
**Pull request by yuting-liu**
In given Docker Engine API, networks are given in Json as the structure of `rx_bytes`, `rx_dropped`, `rx_errors`, `rx_packets`, `tx_bytes`, `tx_dropped`, `tx_errors`, and `tx_packets`.

---
This change is [Reviewable](https://reviewable.io/reviews/docker-java/docker-java/957)

### CODE REVIEW

**Comment by KostyaSha** [comment]
@yuting-liu could you dump json file from live instance and implement serder test?

**Comment by yuting-liu** [comment]
@KostyaSha I'm not quite clear about your suggestions. Would you mind give more details? 
Do you mean that I should add a new `StatisticTest` along with this, or I'm not allowed to add this `StatisticNetWorksConfig`?

**Comment by KostyaSha** [comment]
Something like this https://github.com/docker-java/docker-java/blob/6d9dec35effdefbe53cdde50519fb052d366f3ba/src/test/java/com/github/dockerjava/api/model/ContainerTest.java#L19

Dump with wireshark or curl response, place in resources and roundtrip test for new fields

File: src/test/java/com/github/dockerjava/api/model/StatisticsTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+        final Statistics stat = testRoundTrip(RemoteApiVersion.VERSION_1_27,
+                "containers/container/stats/stats1.json",
+                type
+        );
+
+        assertThat(stat.getRead(), equalTo("2017-12-06T00:42:03.8352972Z"));
+
+        final StatisticNetWorksConfig networkConfig = stat.getNetworks().get("eth0");
+        assertThat(networkConfig.getRxBytes(), equalTo(1230L));
```
Comment (KostyaSha) [inline src/test/java/com/github/dockerjava/api/model/StatisticsTest.java]: you can use is()

**Comment by KostyaSha** [comment]
Thanks!

**Comment by yuting-liu** [comment]
@KostyaSha I have a question about this. Since Docker API will change frequently, do we need to change these classes every time with the API update.

**Pull request merged**